Translate/ContentValueDocumentTranslationAPI.cs

```diff
@@ -32,5 +33,16 @@ public class ContentValueDocumentTranslationAPI
             get;
             set;
         }
+
+        /// <summary>
```

Comment:
Is it possible to make this clearer as it uses the "content value" which the other property is called

Reply:
Done